libraries/helpers.rb

```diff
@@ -600,6 +600,17 @@ def are_queues_updated?
   config["Scheduling"] != previous_config["Scheduling"] or is_compute_node_bootstrap_timeout_updated?(previous_config, config)
 end
 
+def are_mount_or_unmount_require?
+  require 'json'
+  change_set = JSON.load_file("#{node['cluster']['shared_dir']}/change-set.json")
+  change_set["changeSet"].each do |change|
+    next unless change["updatePolicy"] == "SHARED_STORAGE_UPDATE_POLICY"
```

Comment:
So, we are assuming that every time a mount/unmount is requested then the changeset entry **always** contains `SHARED_STORAGE_UPDATE_POLICY`. Is this actually the case?

Reply:
Yes, this is the case. When changeset entry contains `SHARED_STORAGE_UPDATE_POLICY`, it means it will be compute_fleet_stop/queue_update_strategy. (Unsupported case for awsbatch is blocked by validation.) In all these cases unmount/mount is required